packages/icon-library/generate-exports.js

```diff
@@ -0,0 +1,32 @@
+const fs = require('fs')
```

Comment:
This is way less readable than my bash script...

https://github.com/Royal-Navy/standards-toolkit/pull/379/files#diff-d975a43e2deef42acab44c11f3da4fde

You can run the shell script on windows still??

Not sure I agree with this?

Reply:
Just use WSL?

Reply:
WSL is overkill just for 1 script. Plus I used WSL and came away with the impression it's an all or nothing thing with it. The reason is that your ide or git tool will be running using native windows git but in wsl you are using the vm git. The upshot was that if I used vscode the githooks had to work on windows. Why not have a script that is OS agnostic instead of one that is specific to Bash?